core/classes/Integrations/IntegrationBase.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/*
+ *	Made by Partydragen
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.0.0-pr13
+ *
```

Comment:
can you add the `@package ...` tag here, so it shows up in the phpdoc website?
also, if you can use the `@author` and `@version` and `@license` that would be great - since that is how the phpdoc website gets the data about each class